src/window.rs

```diff
@@ -358,13 +358,20 @@ impl Window {
         self.window.set_urgent(is_urgent);
     }
 
+    #[cfg(target_os = "macos")]
+    pub fn set_urgent(&self, is_urgent: bool) {
+        use glutin::os::macos::WindowExt;
+        self.window.request_user_attention(is_urgent);
+    }
+
     #[cfg(
         not(
             any(
                 target_os = "linux",
                 target_os = "freebsd",
                 target_os = "dragonfly",
-                target_os = "openbsd"
+                target_os = "openbsd",
+                target_os = "macos"
```

Comment:
This should be changed to `#[cfg(windows)]`, from the platforms we support, that's the only one this applies to.